### ISSUE TITLE
degression: breaking output change for cloudwatch log subscription filters

### ISSUE THREAD

**Issue by mariux** [KyleRoss/node-lambda-log#11]
commit 5e84b447d1d1a24b10eafb496eb5d2989ac97f8a introduced a change in output format: lambda will not add timestamp and request id prefix anymore.

this leads to cloudwatch log subscription filters fail, because the pattern for lambda functions does not match anymore. and this leads in my case to empty dashboard because the subscription filter is not triggered anymore.

expected output (as of 2.0.0): `TIMESTAMD REQUEST-ID JSON`
output 2.0.1: `JSON`

Filterpattern in cloudwatch subscription: Filter pattern: `[timestamp=*Z, request_id="*-*", event]`

i downgraded to 2.0.0 for now. But since this is lambda-log i actually care about those fields (by default).. how can i fix it in the future? 

i would be happy to have the lambda prefixes by default - but thats just me ;)

thanks

**Comment by KyleRoss** [comment]
@mariux A quick fix for this is to override the custom `Console` instance:

```js
const log = require('lambda-log');

// override the console instance with global console
log.console = console;
```

I'm working on the changes and will hopefully have a new version pushed out in the next day or so.

**Comment by KyleRoss** [comment]
Fixed in version 2.1.0. This release has been published to NPM. Please let me know if you are still facing issues after upgrading.

Thanks for reporting!

**Comment by KyleRoss** [comment]
@all-contributors please add @mariux for bug
